build.sbt

```diff
@@ -1380,7 +1380,6 @@ lazy val `engine-runner` = project
   .dependsOn(pkg)
   .dependsOn(cli)
   .dependsOn(`library-manager`)
-  .dependsOn(`language-server`)
```

Comment:
I'm pretty sure we need this dependency or it won't compile :thinking:

Reply:
Yeah, looks like runner is dependent on it.